utilities_common/portstat.py

```diff
@@ -47,7 +47,7 @@
 The order and count of statistics mentioned below needs to be in sync with the values in portstat script
 So, any fields added/deleted in here should be reflected in portstat script also
 """
-BUCKET_NUM = 50
+BUCKET_NUM = 52
```

Comment:
Could you change this to sum of all the list lengths?

Reply:
@kperumalbfn resolved